project/AppDependencies.scala

```diff
@@ -15,6 +15,7 @@ object AppDependencies {
 
   val test: Seq[ModuleID] = Seq(
     "org.scalatest"               %% "scalatest"                % "3.2.12",
+    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-test-play-28"  %"0.73.0",
     "org.scalatestplus.play"      %% "scalatestplus-play"       % "5.1.0",
     "org.pegdown"                 %  "pegdown"                  % "1.6.0",
     "org.jsoup"                   %  "jsoup"                    % "1.15.1",
```

Comment:
all dependencies here can be updated, I suggest to delete all mockito one and add scalatest-plus.mockito in here only, for scalacheck is the same

Also dont need overrides if you update all libs here

Reply:
done